src/main/java/io/nats/client/api/MessageGetRequest.java

```diff
@@ -53,46 +53,6 @@ public static MessageGetRequest nextForSubject(long sequence, String subject) {
         return new MessageGetRequest(sequence, null, subject, null);
     }
 
-    /**
```

Comment:
These deprecated methods are just moved to the end of the file. They are NOT deleted.